apps/web/modules/auth/lib/brevo.ts

```
@@ -0,0 +1,42 @@
+  validateInputs([id, ZId], [email, ZUserEmail]);
+
+  try {
+    const requestBody: any = {
+      email,
+      ext_id: id,
+      updateEnabled: false,
+    };
+
+    // Add `listIds` only if `BREVO_LIST_ID` is defined
+    const listId = BREVO_LIST_ID ? parseInt(BREVO_LIST_ID, 10) : null;
+    if (listId && !isNaN(listId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)